rest-api/tools/import_organizations.sh

```diff
@@ -36,3 +56,9 @@ fi
 source tools/set_path.sh
 python tools/import_organizations.py --awardee_file data/awardees.csv \
   --organization_file data/organizations.csv --site_file data/sites.csv $DRY_RUN
+
+function finish {
+  cleanup
```

Comment:
I don't think this does what you want it to do...

tools/auth_setup.sh above will override cleanup, so that finish here winds up only calling the version in auth_setup.

You also don't need to call trap cleanup above. There can only be one trap function I believe (which is the reason you have to do anything special here.)

Instead what you want is something like:


\# Intentionally does nothing.
function cleanup {
}

...

function finish {
  cleanup
  rm -rf ...
}

trap finish EXIT

This will make finish run, and call cleanup first thing... which if auth_setup.sh overrode it will actually do something.

Does that make sense?